scripts/compactor_runner.py

```diff
@@ -213,6 +172,9 @@ def get_corfu_compactor_cmd(self, compactor_config, class_to_invoke):
 
         return " ".join(cmd)
 
+    def _print_and_log(self, msg):
```

Comment:
I don't see this used anywhere. Can be removed.

Reply:
Done, Thank you.